scripts/tests/test_keys.py

```diff
@@ -1,3 +1,6 @@
+# Copyright 2024 Denis Mingulov
```

Comment:
I am agree to add the missing copyright notice on my behalf.

Reply:
Thank you for confirming.